### PR TITLE
Keyboard

### PR DESCRIPTION
**Pull request by kenogo**

### CODE REVIEW

File: bspwc/keyboard.c [hunk 1]
```diff
@@ -46,3 +46,8 @@ struct keyboard* create_keyboard(struct input* input, struct wlr_input_device* d
 
 	return keyboard;
 }
+
+void destroy_keyboard(struct keyboard* keyboard)
+{
+	free(keyboard);
```
Comment (kenogo) [inline bspwc/keyboard.c]: For some reason, this causes the error `free(): invalid next size (fast)` when terminating the server. This is an error typically thrown when trying to free a pointer that doesn't point to a place where memory was allocated. But in this case, memory was allocated for the keyboard struct, so I don't know what is going on.

**Comment by Bl4ckb0ne** [comment]
I'm reworking the input system this week. I'll ping you when it's to rebase this branch. 
In the meanwhile, can you adapt the code to the [style guide](https://github.com/Bl4ckb0ne/bspwc/blob/master/CONTRIBUTE.md) please?

**Comment by Bl4ckb0ne** [comment]
@kenogo do you want to add more to this PR or I can start review it ?

**Comment by kenogo** [comment]
Sorry yeah I still need to add something, I'm not sure about how to continue from them, so a review would be helpful then. I'll push in about 7 hours

**Comment by Bl4ckb0ne** [comment]
Thank you for your work!

**Pull request merged**